docs/integrations.md

```diff
@@ -18,5 +18,7 @@ _Integrations allow you to set up access to data and events from our platform in
    - For getting data from our platform by making queries to an SQL database.
 4. [Integration based on using an SDK](/integrations_sdk.html)
    - For getting information by calling functions of an SDK installed in your programming environment.
+5. [Integration based on single_sign_on](/integrations_sso.html)
+   - For accessing our services by logging in with a supported identity provider. 
```

Comment:
For accessing our services by using a registered OAuth application (client credentials and authorization code grants).